### PR TITLE
Wide resnet

### PR DESCRIPTION
**Pull request by makaveli10**
-- add wide resnet

### CODE REVIEW

**Comment by aditya-dl** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # torchtrtz
 
-This project is mainly used t generate tensorrt weights for [makaveli10/cpptensorrtz](https://github.com/makaveli10/cpptensorrtz)
+This project is mainly used to generate tensorrt weights for [makaveli10/cpptensorrtz](https://github.com/makaveli10/cpptensorrtz)
```
Comment (shivamswarnkar) [inline README.md:3]: shouldn't this be a different commit?

Comment (makaveli10) [reply README.md:3]: It's a different commit right? Update: readme ?
There are 2 commits. Am I missing something?

Comment (shivamswarnkar) [reply README.md:3]: Oh sorry, I missed that.
